yarn-project/archiver/src/archiver/archiver_store.ts

```diff
@@ -15,23 +15,148 @@ import { L1ToL2MessageStore, PendingL1ToL2MessageStore } from './l1_to_l2_messag
+   * @returns The requested `unverifiedData`.
+   */
+  getUnverifiedData(from: number, take: number): Promise<EventLogs[]>;
+
+  /**
+   * Gets the `take` amount of unverified data starting from `from`.
+   * @param from - Number of the L2 block to which corresponds the first `unverifiedData` to be returned.
+   * @param take - The number of `unverifiedData` to return.
+   * @returns The requested `unverifiedData`.
+   */
+  getEncryptedEventLogs(from: number, take: number): Promise<EventLogs[]>;
+
+  /**
+   * Store new Contract Public Data from an L2 block to the store's list.
+   * @param data - List of contracts' data to be added.
+   * @param blockNum - Number of the L2 block the contract data was deployed in.
+   * @returns True if the operation is successful.
+   */
   addL2ContractPublicData(data: ContractPublicData[], blockNum: number): Promise<boolean>;
+
+  /**
+   * Lookup the L2 contract data for a contract address.
+   * @param contractAddress - The contract data address.
+   * @returns The contract's public data.
+   */
   getL2ContractPublicData(contractAddress: AztecAddress): Promise<ContractPublicData | undefined>;
+
+  /**
+   * Lookup all contract data in an L2 block.
+   * @param blockNum - The block number to get all contract data from.
+   * @returns All contract public data in the block (if found).
+   */
   getL2ContractPublicDataInBlock(blockNum: number): Promise<ContractPublicData[]>;
+
+  /**
+   * Get basic info for an L2 contract.
+   * Contains contract address & the ethereum portal address.
+   * @param contractAddress - The contract data address.
+   * @returns ContractData with the portal address (if we didn't throw an error).
+   */
   getL2ContractInfo(contractAddress: AztecAddress): Promise<ContractData | undefined>;
+
+  /**
+   * Get basic info for an all L2 contracts deployed in a block.
+   * Contains contract address & the ethereum portal address.
+   * @param l2BlockNum - Number of the L2 block where contracts were deployed.
+   * @returns ContractData with the portal address (if we didn't throw an error).
+   */
   getL2ContractInfoInBlock(l2BlockNum: number): Promise<ContractData[] | undefined>;
+
+  /**
+   * Gets the number of the latest L2 block processed.
+   * @returns The number of the latest L2 block processed.
+   */
   getBlockHeight(): Promise<number>;
+
+  /**
+   * Gets the length of L2 blocks in store.
+   * @returns The length of L2 Blocks stored.
+   */
   getBlocksLength(): number;
+
+  /**
+   * Gets the L2 block number associated with the latest unverified data.
+   * @returns The L2 block number associated with the latest unverified data.
+   */
   getLatestUnverifiedDataBlockNum(): Promise<number>;
+
+  /**
+   * Gets the L2 block number associated with the latest encrypted event logs.
+   * @returns The L2 block number associated with the latest encrypted event logs.
+   */
+  getLatestEncryptedEventLogsBlockNum(): Promise<number>;
```

Comment:
Please rename to `getLatestEncryptedLogsBlockNum` as well as the other functions which contain "EventLog".